open-hackathon-adminUI/src/app/config.py

```diff
@@ -47,8 +51,8 @@
             "response_type": "response_type=code",
         },
         "hackathon": {
-            "endpoint": HOSTNAME + ":15000",
-            "local_endpoint": HOSTNAME + ":80"
+            "name": "open-xml-sdk",
+            "endpoint": HACkATHON_API_ENDPOINT
         }
     }
 }
```

Comment:
the `config.py` shouldn't be involved in commits. Push `config-sample.py` only to github